lib/hyrax/configuration.rb

```diff
@@ -506,6 +506,13 @@ def default_nested_relationship_reindexer
       ->(id:, extent:) { Samvera::NestingIndexer.reindex_relationships(id: id, extent: extent) }
     end
 
+    # @return [integer] max # of collections to return in query
+    # Max number of collections, to be used when we need to return all in a solr query
+    attr_writer :collection_query_limit
+    def collection_query_limit
+      @collection_query_limit ||= Collection.count + 1_000
+    end
+
```

Comment:
I would have expected this to use the value from the initializer config/initializers/hyrax.rb `config.collection_query_limit`.

Reply:
I confirmed that the configuration set in `config/initializers/hyrax.rb` is honored.  So this is fine.